### PR TITLE
update deposit modal

### PR DESCRIPTION
**Pull request by daiki44**
- fix style
- add url parameter
スクリーンショット 2020-05-07 19 59 56
スクリーンショット 2020-05-07 20 01 39
スクリーンショット 2020-05-07 20 01 45

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -13,11 +13,11 @@ export const NFT_COLLECTIBLES = '/nft_collectibles'
  * @name openModal
  * @param {*} modalName is "deposit" or "withdraw"
  */
-export function openModal(modalName) {
+export function openModal(modalName, token) {
   Router.push(
     {
       pathname: Router.pathname,
-      query: { modal: modalName }
+      query: { modal: modalName, token: token }
```
Comment (tkmct) [inline src/routes.js:20]: you can use shorthand for object value with same name with the key

File: src/routes.js [hunk 2]
```diff
@@ -13,11 +13,11 @@ export const NFT_COLLECTIBLES = '/nft_collectibles'
  * @name openModal
  * @param {*} modalName is "deposit" or "withdraw"
  */
-export function openModal(modalName) {
+export function openModal(modalName, token) {
```
Comment (tkmct) [inline src/routes.js:16]: I think token is too specific for the modal argument.
what about just add `args`?

**Comment by tkmct** [comment]
Doesn't the border-radius of amount-input too small?

**Comment by daiki44** [comment]
> Doesn't the border-radius of amount-input too small?
@tkmct What do you think about this? 4px -> 8px
スクリーンショット 2020-05-08 21 45 39

File: src/routes.js (hunk 2 above)
Comment (daiki44) [reply src/routes.js]: like this?
https://github.com/cryptoeconomicslab/WakkanayBrowserWallet/blob/94d56317846a29e167a8b8e6b50deb5dff8d001f/src/routes.js#L18

**Comment by daiki44** [comment]
updated!

**Comment by tkmct** [comment]
For the border-radius, please refer to the Figma.
The border-radius value is 20px.

**Comment by daiki44** [comment]
@tkmct updated!

**Pull request merged**